pager.c

```diff
@@ -1339,7 +1339,7 @@ static int format_line (struct line_t **lineInfo, int n, unsigned char *buf,
     {
       if (col + 1 > wrap_cols)
 	break;
-      ++col;
+      col += k;
```

Comment:
@toogley a refactoring of pager.c could be in order - but let's do that elsewhere. It's a particular bug I'm trying to get caught here :)